packages/mdc-top-app-bar/fixed/foundation.ts

```diff
@@ -31,17 +30,10 @@ export class MDCFixedTopAppBarFoundation extends MDCTopAppBarFoundation {
    */
   private wasScrolled_ = false;
 
-  /* istanbul ignore next: optional argument is not a branch statement */
-  constructor(adapter?: Partial<MDCTopAppBarAdapter>) {
-    super(adapter);
-
-    this.scrollHandler_ = () => this.fixedScrollHandler_();
-  }
-
   /**
    * Scroll handler for applying/removing the modifier class on the fixed top app bar.
    */
-  private fixedScrollHandler_() {
+  handleTargetScroll() {
```

Comment:
Add `@override` annotation all `handleTargetScroll` and `handleWindowScroll` methods in subclasses.

Reply:
I don't see @override anywhere else. Is that a JSDoc thing?